src/mobile-token/tokenService.ts

```diff
@@ -201,6 +214,7 @@ const service: TokenService = {
             },
             authWithIdToken: true,
             timeout: 15000,
+            // skipErrorLogging: (err) => isGrpcErrorType(err.response?.data),
```

Comment:
```suggestion
```

Reply:
Opps. Should actually be like this:
```suggestion
            skipErrorLogging: (err) => isGrpcErrorType(err.response?.data),
```